solutions/security-enforced/README.md

```diff
@@ -0,0 +1,82 @@
+# Event Notifications solution
+
+When `existing_en_instance_crn` is not passed, this solution configures the following infrastructure:
+
+- optionally a KMS key ring and key for IBM Event Notifications encryption
+- optionally a KMS key ring and key for IBM Cloud Object Storage encryption
```

Comment:
KMS is mandated in security enforced. So I would not use the word "optionally" here. The options are the DA either creates the key and key ring, or the DA takes in existing key